tests/blend.py

```diff
@@ -64,164 +74,151 @@ def colors(request):
 
 
 @pytest.fixture
-def f32_channel_low_med_high():
-    return np.float32([
-        [0.0],
-        [256.0 / 65535.0],
-        [1.0],
-    ])
+def u16_low_med_high(full_u8, full_u16):
+    return np.array([[0], [full_u8], [full_u16]], dtype=np.uint16)
 
 
 @pytest.fixture
-def channel_low_med_high():
-    return np.uint16([[0], [256], [65535]])
+def f32_low_med_high(full_u8, full_u16):
+    return np.array([[0], [full_u8 / full_u16], [1]], dtype=np.float32)
 
 
 @pytest.fixture
-def channel_check():
-    return np.uint16([
-        [0, 65535],
-        [65535, 0]
-    ])
+def channel_check(full_u16):
```

Comment:
Let's call this checkered as `check` in testing is confusing.

Reply:
done in `refactor names and revert to version 0.0.1`

Reply:
https://github.com/sorgerlab/minerva-lib-python/pull/17/commits/29ed61d369e2d506ea15686cd45fd49e12580ee1#diff-4438daa1ceea9cdeeef65ab8cfb5075dR87